src/components/HomePage.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+import Container from "react-bootstrap/Container";
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+
+function HomePage() {
+  return (
+    <div className="App d-flex  justify-content-center align-items-center">
```

Comment:
Do we need `justify-content-center` here? Doesn't `Container` component handle this functionality?
Reference: https://css-tricks.com/snippets/css/a-guide-to-flexbox/